### PR TITLE
Add checkstyle rules about imports and empty lines between members

### PR DESCRIPTION
**Pull request by leventov**

### CODE REVIEW

**Comment by gianm** [comment]
Are the new checkstyle rules something that the IntelliJ codestyle in the Druid repo can apply automatically? (I feel this is an important question to ask for any new checkstyle rules)

**Comment by leventov** [comment]
Druid's code style is mostly not described anywhere and is largely defined by the IntelliJ's defaults. However, I updated Eclipse import order (but didn't test it), thanks for note.

**Comment by leventov** [comment]
This PR has an approve from @QiuMM who is now a committer, so I merge.

File: pom.xml [hunk 1]
```diff
@@ -878,9 +878,9 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-checkstyle-plugin</artifactId>
-                <version>2.17</version>
+                <version>3.0.0</version>
                 <configuration>
-                    <sourceDirectory>${project.build.sourceDirectory}</sourceDirectory>
+                    <sourceDirectories>${project.build.sourceDirectory}</sourceDirectories>
```
Comment (jihoonson) [inline pom.xml]: @leventov would you tell me why this change is needed? Apparently, `sourceDirectories` needs a list which looks related to https://github.com/apache/incubator-druid/issues/6668.

Comment (leventov) [reply pom.xml]: https://maven.apache.org/plugins/maven-checkstyle-plugin/check-mojo.html#sourceDirectory

Comment (jihoonson) [reply pom.xml]: Thanks.
